src/structs/Webhook.ts

```diff
@@ -28,11 +28,11 @@ export class Webhook {
    * Create a new webhook client instance
    * @param authorization Webhook authorization to verify requests
    */
-  constructor (authorization: string) {
+  constructor (authorization?: string) {
     this.auth = authorization
   }
 
-  private _verifyRequest (req, res): Promise<Vote|false> {
+  private _parseRequest (req, res): Promise<WebhookPayload|false> {
     return new Promise(resolve => {
       if (this.auth && req.headers.authorization !== this.auth) return res.status(403).json({ error: 'Unauthorized', from: 'topggjs' })
```

Comment:
I just realized `from: 'topggjs'` could potentially leak information about the backend or the libraries they're using. Best if we just omit the entire body.